pkg/tlsf/.gitattributes

```diff
@@ -1,2 +0,0 @@
-# Ignore whitespace errors inside these patch files by regarding them as binary.
-patch.txt binary
```

Comment:
A seperate repository does not help me if I want to re-review changes to the package (i.e. the patch files) in the RIOT-repository. I don't see it as spam, but useful information, especially if you need to bisect behavior. For other situations unneeded output is easily to deactivate via git-diffs comprehensive option set ;-).